api/receiver/receive_results_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"google.golang.org/appengine/taskqueue"
 )
 
+// regexMatcher is a gomock.Matcher that verifies whether a string argument
```

Comment:
nit: Rather than emphasizing _what_ regexMatcher is, let's have this comment focus on _why_ it's here. Give an example or pattern in this test file where a matcher that matches a regexp is the right verification tool for these tests.